quickstart/101-azure-virtual-desktop-configure/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "resource_group_name_prefix" {
+  default       = "rg"
+  description   = "Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription."
+}
+
+variable "resource_group_location" {
+  default       = "eastus"
+  description   = "Location of the resource group."
+}
+
+variable "workspace" {
+  type          = string
+  description   = "Name of the Azure Virtual Desktop workspace"
+  default       = "AVD TF Workspace"
+}
+
+variable "deploy_location" {
```

Comment:
This name is a bit ambiguous as well. Can it be clarified? Also, why do we have two different locations?